### ISSUE TITLE
[Suggestion] Reactor sizing on handheld weapons

### ISSUE THREAD

**Issue by ProPeach** [tsunamayo/Starship-EVO#4347]
Loving the new handhelds, especially the smaller scaling!
![image](https://user-images.githubusercontent.com/27857711/145411213-7622d4f3-cf05-4eaa-ab29-6c5b8558d542.png)

One thing that feels odd though, is the scaling of the reactors.
Because their smallest scaling setting is 1/4, you cannot realistically fit a reactor to power your gun.
![image](https://user-images.githubusercontent.com/27857711/145411421-29522c7f-d3ef-4682-82c3-0f1b6cd434eb.png)
![image](https://user-images.githubusercontent.com/27857711/145411513-8d4807b1-cd7e-4764-9030-6100d36edf37.png)

Would it be possible to have a 1/8th scaling option for reactors, but only for handhelds? I like the 1/4th scaling option for ships and it makes sense, but it doesn't quite work for the same reasons on handhelds

**Comment by ExodistSKY1** [comment]
I too agree with Peach. Having the addition of the 1/8th reactors solely for handhelds would be ideal. and a greatly welcomed addition by the community as a whole.

**Comment by ProPeach** [comment]
An alternative would be for the handle to provide as much power as say, half a 1/4th scale reactor. It would be pretty weird and need some good documentation to show players, but might help out. It also scales quite nicely as 2 handed guns tend to need more power, and will get extra from 2 handles

**Comment by cloudrambler** [comment]
What about having a dedicated weapon battery? This would not require any fuel, but instead would charge when the weapon was on a mount. The battery could be any shape that is appropriate for a weapon aesthetic.

Having a powered handle could work on the same basis (inherently containing a small battery).

**Comment by tsunamayo** [comment]
yeah honestly there is a lot of things wrong with handheld, I dont have much time for them atm... I think maybe we could reduce the size reduction factor even more from 0.5 to 0.25, but I would have to prevent building things like button or that would look weird.

**Comment by ProPeach** [comment]
Yeah, I guess that would work too! It'd be the simplest option here for sure

**Comment by ZachZent** [comment]
> 

Though I wouldn't be against another size reduction, I think it is best not to design handheld system around a ships reactor core. It would be best to wait for whatever future weapons update and instead use batteries or actual ammo mags/clips for ammo instead of plugging in a ship's reactor core
